docs/user-manual/en/cdi-integration.md

```diff
@@ -0,0 +1,32 @@
+# CDI Integration
+
+Apache ActiveMQ Artemis provides a simple CDI integration.  It can either use an embedded broker or connect to a remote broker.
+
+## Configuring a connection
+
+Configuration is provided by implementing the `ArtemisClientConfiguration` interface.
+
+```java
+public interface ArtemisClientConfiguration {
+   String getHost();
+
+   Integer getPort();
+
+   String getUsername();
+
+   String getPassword();
+
+   String getUrl();
+
+   String getConnectorFactory();
+
+   boolean startEmbeddedBroker();
+
+   boolean isHa();
+
+   boolean hasAuthentication();
+}
+```
+
+There's a default configuration out of the box, if none is specified.  This will generate an embedded broker.
```

Comment:
Can we improve the doc a little here? how to use it.. etc? just a paragraph or two is fine.

I will merge it regardless of this.. but it would be nice to improve it.